main.lua

```diff
@@ -76,6 +103,9 @@ printResults = function (results, level)
 end
 
 lib.load = function ()
+  -- Write the header of the README.md file.
+  io.write(header)
+
```

Comment:
Writing the header is not the only thing to do for generating the README file in the markdown format. Check how this one looks on github for this branch and you'll see the statuses are not displayed. You should update the printing function to use markdown formatting (by inserting characters so that it is displayed as lists for instance).